### PR TITLE
[wasm][debugging] Enable debugging when Device Toolbar is enabled and Iphone is selected

### PR DESCRIPTION
**Pull request by thaystg**
When DevTools is enabled and iPhone 12 is selected, navigatorUA.userAgentData.brands has length = 0 and Debugger was disabled.
With this PR is possible to debug when Device iPhone is selected.
image

Fixes:
https://github.com/dotnet/runtime/issues/84171#issuecomment-1567730864

### CODE REVIEW

**Review by kg — APPROVED** [review]

**Comment by pavelsavara** [comment]
We have similar test in `src\mono\wasm\runtime\scheduling.ts` which uses it to prevent CPU throttling. Could you please also fix that or unify it.

**Comment by thaystg** [comment]
> scheduling.ts

As discussed with you, you will work on it.

File: src/mono/wasm/runtime/loader/polyfills.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 
 import type { DotnetModuleInternal } from "../types/internal";
-import { INTERNAL, ENVIRONMENT_IS_NODE, ENVIRONMENT_IS_SHELL, loaderHelpers } from "./globals";
+import { INTERNAL, ENVIRONMENT_IS_NODE, ENVIRONMENT_IS_SHELL, loaderHelpers, ENVIRONMENT_IS_WEB } from "./globals";
 
 let node_fs: any | undefined = undefined;
 let node_url: any | undefined = undefined;
 
-export async function init_polyfills(module: DotnetModuleInternal): Promise<void> {
+export async function detect_features_and_polyfill(module: DotnetModuleInternal): Promise<void> {
```
Comment (pavelsavara) [inline src/mono/wasm/runtime/loader/polyfills.ts:8]: @kg we could start adding more feature detection here on next PRs, like SIMD, MT etc

**Comment by pavelsavara** [comment]
image

**Pull request merged**